core/src/test/java/org/sonarsource/sonarlint/core/mediumtest/ConnectedIssueExclusionsMediumTests.java

```diff
@@ -82,24 +81,23 @@ public static void prepare() throws Exception {
       .addEnabledLanguage(Language.JAVA)
       .build();
     sonarlint = new ConnectedSonarLintEngineImpl(config);
-    baseDir = temp.newFolder();
   }
 
-  @AfterClass
+  @AfterAll
   public static void stop() {
     if (sonarlint != null) {
       sonarlint.stop(true);
       sonarlint = null;
     }
   }
 
-  @Before
+  @BeforeEach
```

Comment:
Do we need to keep public visibility on @BeforeEach and related ? There are several occurrences